server/config/config.go

```diff
@@ -911,6 +908,14 @@ func (c *ScheduleConfig) migrateConfigurationMap() map[string][2]*bool {
 	}
 }
 
+// GetMaxMergeRegionKeys returns the max merge keys.
+func (c *ScheduleConfig) GetMaxMergeRegionKeys() uint64 {
+	if keys := c.MaxMergeRegionKeys; keys != 0 {
+		return keys
+	}
+	return c.MaxMergeRegionSize * 10000
```

Comment:
10000？

Reply:
yes， the value is same with tikv: https://github.com/tikv/tikv/blob/f7edbcf610cd6bb5c9040317edd3260188a6e87d/components/raftstore/src/coprocessor/config.rs#L102-L106

Reply:
Could you please help add a comment to tell this is to keep consistent with TiKV?

Reply:
fix